### PR TITLE
homePage first view

### PR DESCRIPTION
**Pull request by DamianTobiasz**
Problems with ng serve

### CODE REVIEW

File: projects/home/src/lib/adapters/primary/ui/today.component.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="py-3">
+  <p class="px-3 fs-1 fw-bold d-inline align-middle text-uppercase">Today</p>
+  <p class="px-3 fs-5 d-inline align-middle text-uppercase">Fri 18 MAR</p>
```
Comment (gradzio) [inline projects/home/src/lib/adapters/primary/ui/today.component.html]: Please use date object to always show todays date

**Review by gradzio — CHANGES_REQUESTED** [review]

**Comment by DamianTobiasz** [comment]
Updated view

File: projects/home/src/lib/adapters/primary/ui/images-inline.component.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { ImageDTO } from '../../../application/ports/secondary/image.dto';
+
+@Component({
+  selector: 'lib-images-inline',
+  templateUrl: './images-inline.component.html',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ImagesInlineComponent {
+  images$: Observable<ImageDTO[]> = of([
```
Comment (gradzio) [inline projects/home/src/lib/adapters/primary/ui/images-inline.component.ts:16]: It should be in the service and fetched via port

File: projects/home/src/lib/adapters/primary/ui/today.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+} from '@angular/core';
+
+@Component({
+  selector: 'lib-today',
+  templateUrl: './today.component.html',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TodayComponent {
+  today: Date = new Date();
```
Comment (gradzio) [inline projects/home/src/lib/adapters/primary/ui/today.component.ts:14]: It should be dto

File: src/assets/customStyles.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.violet {
```
Comment (gradzio) [inline src/assets/customStyles.scss:1]: Please customize bootstrap colors

File: src/assets/customStyles.scss [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  background-color: #7749f8;
+}
+
+.base-pad {
+  padding-left: 165px;
+  padding-right: 165px;
+}
+
+.img-size {
+  width: 349px;
```
Comment (gradzio) [inline src/assets/customStyles.scss:11]: Use responsivr classes

**Review by gradzio — CHANGES_REQUESTED** [review]

**Pull request merged**